### PR TITLE
Added assertion for web on AudioCache and Documentation

### PR DESCRIPTION
**Pull request by BrayFlex**

### CODE REVIEW

File: lib/audio_cache.dart [hunk 1]
```diff
@@ -37,7 +37,8 @@ class AudioCache {
   bool respectSilence;
 
   AudioCache(
-      {this.prefix = "assets/", this.fixedPlayer, this.respectSilence = false});
+      {this.prefix = "assets/", this.fixedPlayer, this.respectSilence = false})
+       : assert(kIsWeb != true, 'AudioCache is not available for Flutter Web.');
```
Comment (erickzanardo) [inline lib/audio_cache.dart:41]: Suggestion: `!kIsWeb` instead of `kIsWeb != true`

**Review by erickzanardo — APPROVED** [review]
One minor suggestion, but LGTM already

**Review by luanpotter — DISMISSED** [review]
LGTM

**Comment by luanpotter** [comment]
@BrayFlex please fix formatting so I can merge

**Review by erickzanardo — APPROVED** [review]
LGTM

**Review by luanpotter — APPROVED** [review]

**Pull request merged**